testing/test_skipping.py

```diff
@@ -382,7 +382,82 @@ def test_func():
         ])
 
 
-class TestSkipif:
+class TestSkip(object):
+    def test_skip_class(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip
+            class TestSomething(object):
+                def test_foo(self):
+                    pass
+                def test_bar(self):
+                    pass
+
+            def test_baz():
+                pass
+        """)
+        rec = testdir.inline_run()
+        rec.assertoutcome(skipped=2, passed=1)
+
+    def test_skips_on_false_string(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip('False')
+            def test_foo():
+                pass
+        """)
+        rec = testdir.inline_run()
+        rec.assertoutcome(skipped=1)
+
+    def test_skip_no_reason(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip
+            def test_foo():
```

Comment:
Just to be clear here @MichaelAquilina, I think the message should be improved a little. This is what I see with a `pytest.mark.skip` without any parameters:

```
SKIP [1] skips.py:2: <Skipped instance>
```

IMO this should be something along the lines of `"unconditional skip"` or `"marked for skipping"`.